### ISSUE TITLE
Critical errors and Warnings in QGIS 3.4

### ISSUE THREAD

**Issue by kingwarrick** [alexbruy/processing-whitebox#12]
Ignore this, working with WhiteboxGAT 3.4, whereas plugin states support for 0.8.0.

**Comment by jblindsay** [comment]
@kingwarrick I just wanted to clarify that Whitebox GAT and WhiteboxTools are different projects. processing-whitebox uses WhiteboxTools, which is currently at version 0.11 (soon to be 0.12).

**Comment by kingwarrick** [comment]
Thanks @jblindsay, the clarification is appreciated. I will grab WhiteboxTools instead.

**Comment by alexbruy** [comment]
Please note that plugin version and WhiteboxTools version are not the same versions: while latest plugin version is 0.8 it supports WhiteboxTools 0.11

**Comment by jblindsay** [comment]
@kingwarrick you're welcome. I already regret naming the two projects so similarly. When I started the WhiteboxTools project I envisioned it mainly as serving as a new back-end for Whitebox GAT, but increasingly I see it more as a generic geospatial analysis library that can be embedded in other front-ends, including QGIS. If I could go back, I would have named it very differently. Perhaps I should start to refer to it more as WBT instead.
